### DISCUSSION TITLE
Trajectory analysis with CMAP corrections in force field

### DISCUSSION THREAD

**Discussion by AnsariMF** [MDAnalysis/mdanalysis#4801] [Q&A]
Hello,
I am working on IDPs. As their are so many advanced force fields developed. I am using that force field for IDP simulation. Now I want to analyze some part of the trajectory with MDAnalysis. But whenever i am giving topology file to read, it shows me error File "/apps/anaconda3/lib/python3.12/site-packages/MDAnalysis/core/universe.py", line 110, in _topology_from_file_like
    topology = p.parse(**kwargs)
               ^^^^^^^^^^^^^^^^^
  File "/apps/anaconda3/lib/python3.12/site-packages/MDAnalysis/topology/TOPParser.py", line 226, in parse
    raise ValueError(emsg)
ValueError: top_prot_wat.prmtop is detected as a Chamber-style TOP file. At this time MDAnalysis does not support such topologies

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home2/munazzah/Project2/Methods/GaMD/Mix_traj/c_Myb/6.Analysis/KD/test4.py", line 13, in <module>
    u = mda.Universe("top_prot_wat.prmtop", "gamd.m.nc")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/apps/anaconda3/lib/python3.12/site-packages/MDAnalysis/core/universe.py", line 356, in __init__
    topology = _topology_from_file_like(self.filename,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/apps/anaconda3/lib/python3.12/site-packages/MDAnalysis/core/universe.py", line 125, in _topology_from_file_like
    raise ValueError(
ValueError: Failed to construct topology from file top_prot_wat.prmtop with parser <class 'MDAnalysis.topology.TOPParser.TOPParser'>.
Error: top_prot_wat.prmtop is detected as a Chamber-style TOP file. At this time MDAnalysis does not support such topologies . Is the MDAnalysis module not compatible with this type of topology? Is there some other way to study this type of topology file? 

Thanks
Munazzah Ansari
PhD student

**Comment by orbeckst** [comment]
Unfortunately, these files are (still) not supported (see issue #2475 )

There's a work-around using ParmEd as posted in https://github.com/MDAnalysis/mdanalysis/issues/2475#issuecomment-1303822464 — see if that helps.

Please feel free to post your error on the issue. That would show that users are interested in seeing it fixed and then perhaps someone volunteers to take it on.

**Reply by AnsariMF** [reply]
Can you give suggestions to handle this type of file in MDAnalysis or with any other package I can handle and do analysis with MDAnalysis ? I want to do an ensemble comparison study with different ensembles.

**Reply by IAlibay** [reply]
@AnsariMF the issue is that chamber-style PRMTOPs don't adhere to the PRMTOP standard and are more of an extension created by Parmed (as far as I remember).

I haven't tried, but I think the easiest way to deal with this would be to load the topology using parmed and then use the parmed converter to ingest the topology into an MDAnalysis universe. From then, you could then load in the coordinates.

**Reply by AnsariMF** [reply]
Thanks for the suggestion. I will try this.
